copter/source/docs/traditional-helicopter-rsc-setup.rst

```diff
@@ -4,16 +4,32 @@
 Rotor Speed Control Setup
 =========================
 
-Rotor speed controller set up is described in video 3
-for electric powered helicopters and video 4 for piston 
-powered helicopters on the following page.
+The Rotor Speed Control (RSC) uses the throttle output to the engine to control rotor speed. This can be done in a few different ways either through an open loop control called a throttle curve, a set point signal to an external governor or electronic speed controller, or a closed loop internal governor. The RSC also controls the throttle for ground idle, rotor run up and shut down. This wiki covers the set up of each of the parameters in the RSC.
 
-:ref:`traditional-helicopter-configuration`
+RSC mode
+========
 
+First, set the RSC Mode parameter (:ref:`H_RSC_MODE <H_RSC_MODE>`). The RSC modes are listed below with a short description. 
+
+RC Passthrough - this mode passes through the RC channel input on which the motor interlock is assigned. 
+
+..warning::
+    Setting the RSC mode to RC passthrough requires configuring the RC receiver to hold last value for the Motor Interlock channel (default is channel 8). If the receiver loses connection to the transmitter and receiver is not configured correctly, the motor will shutdown and the helicopter will crash!
+
+RSC setpoint - this mode is used for helicopters utilizing either an electronic speed controller or an external governor for internal combustion engines. The PWM passed to the Heli RSC channel is determined from the External Motor Governor Setpoint (:ref:`H_RSC_SETPOINT <H_RSC_SETPOINT>`) parameter. The output PWM is calculated by the following equation:
+PWM output = RSC_SETPOINT * 0.01 * (SERVO_MAX - SERVO_MIN) + SERVO_MIN
```

Comment:
@bnsgeyer I don't know that the actual code equation would have to be displayed in the wiki. IMO the average user doesn't care about that.

Reply:
I think it was worth showing it in case they were trying to match an output value for their ESC.  Like setting 30 %, 70% and 100% for different rotor speeds.